.travis.yml

```diff
@@ -1,20 +1,16 @@
 language: python
 # dist: xenial  # required for Python >= 3.7 (travis-ci/travis-ci#9069)
-python:
-  - 2.7
-  - 3.6
-  - 3.7
-env:
-  - TOXENV=quality
-  
+
 matrix:
```

Comment:
If you're running quality and tests on both python 2 and 3, I don't think you need to add matrix.

Reply:
> If you're running quality and tests on both python 2 and 3, I don't think you need to add matrix.

I added the matrix because I needed different python environments for both versions i.e. py27 for Python 2.7 and py37 for Python 3.7.

Reply:
I don't understand, why is that necessary?